### ISSUE TITLE
Make a "Quick Addon Settings" panel module. Highly inspired by MBB - Minimap Button Bag.

### ISSUE THREAD

**Issue by Cruzix** [shagu/pfUI#218]
This module should make you able to access all your other addons' settings through a pop out menu on either mouseover or mouse click, very similar to [MBB - Minimap Button Bag's](http://addons.us.to/addon/mbb) functionality.
However instead of gathering the addon minimap buttons and grouping them in a menu like MBB does, 
it should remove/hide all minimap buttons (optional) and then make you able to add addons to the list by pressing "add addon" on top of the list where you then have to fill out 2 things:

_Example:_
Addon Name: TrinketMenu
Addon Command: /Trinketmenu options

The module will then add "TrinketMenu" to the list and when you click on it in the list, it will execute /Trinketmenu Options. This will make you able to get a clean looking minimap without ugly buttons everywhere around it and a place to easily browse your other addons' settings without having to memorize all the commands.

**Comment by Cruzix** [comment]
Here's an addon that hides minimap buttons, but its for 4.3.0 and later patches but maybe you can use its code to hide the buttons if needed https://wow.curseforge.com/projects/no-minimap-buttons/files

**Comment by Road-block** [comment]
I just want to comment on this before shagu considers it, in vanilla addon authors didn't have a standardized way to create minimap buttons, so it is very hard to have an "all-in-one" solution that collects them.  
This is the reason MBB is buggy, DetachedMinimapButtons is buggy and every single addon that tries to provide this functionality is buggy.  
After some point in the life of WoW pretty much everyone relied on fubar / ldb and their included minimap icon libraries to do the job so it was much easier to find and collect minimap buttons.  

TL;DR: This is harder (as in more time consuming) than you might think.

**Comment by Cruzix** [comment]
Is this also an issue if you want to hide them completely rather than repositioning them? (which is what MBB does)

**Comment by shagu** [comment]
It's just too much work. This should be done by a separate addon.
